src/test/java/jnr/constants/DefinedTest.java

```diff
@@ -57,4 +57,22 @@ public void testLangInfoUnix() throws Throwable {
             Assert.assertTrue(LangInfo.CODESET.defined());
         }
     }
+
+    @Test
+    public void testUndefinedConstant() throws Throwable {
+        if (Platform.OS.equals("linux")) {
+            Assert.assertFalse(AddressFamily.AF_CHAOS.defined());
+            if (Platform.FAKE) {
```

Comment:
Do you need to test on Travis with and without this setting then? Or could you mock the setting?